source/guide/3.2/ui-blueprint-upload.md

```diff
@@ -36,14 +36,15 @@ The upload blueprint button can be found in the "Blueprints" section in the UI:
 ![The blueprint upload button](/guide/images/ui/ui_upload_blueprint_button.png)
 
 
-Once the blueprint upload dialog appears, the user can select a blueprint TAR file from the filesystem by pressing the `+` button:
+Clicking on it will cause the blueprint upload dialog to appear. <br>
+The user can either type in the path to the blueprint TAR file, or select it from the filesystem by pressing the `+` button:
 
-![The blueprint upload dialog](/guide/images/ui/ui_upload_dialog.png)
+![The blueprint upload dialog](/guide/images/ui/ui-upload-blueprint.png)
 
+The blueprint name is a required field. The user can also provide a custom blueprint YAML file name. If none is provided, the default `blueprint.yaml` file will be used.<br>
```

Comment:
Nir,

I think you got confused between the blueprint name (which is required) and the blueprint yaml file (which is optional). 

if no file name is provided, the default file name 'blueprint.yaml' file will be used (or expected to exist in the uploaded file).

Maybe 'expected to exist in the uploaded file' is a better way of putting it?